CTFd/themes/admin/templates/config.html

```diff
@@ -105,6 +116,16 @@ <h1>Configuration</h1>
 				{% include "admin/configs/backup.html" %}
 
 				{% include "admin/configs/usermode.html" %}
+
+				{% if plugins %}
+					{% for plugin in plugins %}
+						{% if plugin.config %}
+							<div role="tabpanel" class="tab-pane config-section" id="{{plugin.name  | replace(' ', '')}}">
+								{% include plugin.config %}
```

Comment:
This flow implies that the plugins themselves need to register a template to the jinja loader. How are you proposing that this should be done? 

You can do it as part of a blueprint registration but is it intended that the plugin developer needs to register this template?

Also there's only one namespace so I believe you can't have multiple config files with the same name (i.e. `config_plugin.html` instead of just `config.html`)

Reply:
Plug-in templates are always available as `plugins/plugin_name/path_to_plugin.html`

So a template in the plug-in folder for `mysuperplugin` named `mytemplate.html` would be `plugins/mysuperplugin/mytemplate.html`

Reply:
Ah so it is, I forgot about that loader. Okay that was my major concern with this PR.